src/en/reference-status.md

```diff
@@ -8,18 +8,18 @@ following:
-|waiting      |yellow         | The unit is unable to progress to an active state because a service to which it is related is not running. That service might be in error, blocked, waiting or maintenance. This unit expects it will be able to proceed towards active as soon as the things it is concerned about are resolved there - it needs no human attention (otherwise its status would be blocked).|
-|active |green| This unit believes it is correctly offering all the services it is primarily installed to provide. A service can be active even if additional relations are being configured and set up. For example, a Ceph OSD service that is connected to its monitors is active, even if you then relate it to Nagios; setting up a  subordinate does not impinge on the OSD functionality which is its primary purpose for existing.|
+| blocked      | blue           | This unit needs manual intervention because of a problem associated directly with the unit itself. This may be due to bad or inconsistent configuration of the application, disk space being unavailable for the unit, or missing relations which are essential to application operations. The message should tell a human administrator what is required to unblock the application.<br> For example: “Please provide an object store. Please configure the minimum speed required by users of this application. Please link this application to a database.”|
+|maintenance  |yellow          |The unit is not yet providing applications, but is actively doing stuff in preparation for providing those applications. This is a “spinning” state, not an error state. It reflects activity on the unit itself, not on peers or related units. The Juju agent will set the unit to maintenance when it begins the process of installing the charm - downloading it, unpacking it and calling the install hook.<br>The unit may transition to the maintenance state at any time, in any hook. The maintenance state implies that the unit has no reason to think that it will not provide applications soon. For example, the software might be formatting block devices, or replicating data from peer units, or handling an action which for whatever reason means that it is not actively providing application, but it is not aware of anything on its own config or in related unit config that will prevent it from bringing up its applications.<br>Counterparts to this unit should expect that application calls to this unit will fail during the period it is in the maintenance state, but unless they have an urgent need to move to another counterpart, they should just wait for this unit to finish churning.<br> Being in maintenance does not require human intervention.|
```

Comment:
some of these uses are a bit strained now, but particularly "is not actively providing application" definitely shouldbe changed. I suggest "providing the application".

This is the fun you can have when you swap something which can be a noun or a verb for something which only works as a noun ;)